docs/source/install/ansible.rst

```diff
@@ -0,0 +1,29 @@
+Ansible Playbooks
+=================
+
+Ansible playbooks to install |st2|.
+
+Allows you to deploy and further configure |st2| installation on local or remote machines with Ansible configuration management tool.
+Playbooks source code is available as GitHub repository `ansible-st2 <https://github.com/StackStorm/ansible-st2>`_.
+
+---------------------------
+
+Supported platforms
+---------------------------
+* Ubuntu Trusty (14.04)
+* Ubuntu Xenial (16.04)
+* RHEL6 / CentOS6
+* RHEL7 / CentOS7
+
+Quick Start
+---------------------------
+.. sourcecode:: bash
+
+    git clone https://github.com/StackStorm/ansible-st2.git
+    cd ansible-st2
+
+    ansible-playbook stackstorm.yml
+
+
+Please refer to https://github.com/StackStorm/ansible-st2 for updates and more detailed examples, descriptions and code.
```

Comment:
Bare links get rendered as a link - see https://docs.stackstorm.com/mistral_yaql.html?highlight=yaql#testing-yaql-expressions for an example. In the RST it appears as http://yaqluator.com/., and it gets rendered as a link